sdk/tests/transaction.rs

```diff
@@ -175,7 +190,7 @@ async fn transaction_with_throw_and_return() -> Result<(), Error> {
 	let dbs = new_ds().await?;
 	let ses = Session::owner().with_ns("test").with_db("test");
 	let res = &mut dbs.execute(sql, &ses, None).await?;
-	assert_eq!(res.len(), 1);
+	assert_eq!(res.len(), 4);
```

Comment:
If a return statement gets canceled in a transaction it now no longer causes the transaction to return only a single result.